prisma/schema.sql

```diff
@@ -90,17 +93,4 @@ CREATE TABLE adjudications (
   FOREIGN KEY(performance_id) REFERENCES performances(id)
 );
 
--- Create contacts table
-CREATE TABLE contacts (
-  id SERIAL PRIMARY KEY NOT NULL,
-  school_id INTEGER NOT NULL,
-  year INTEGER NOT NULL,
-  name VARCHAR(255) NOT NULL,
-  email VARCHAR(255) NOT NULL,
-  phone VARCHAR(255),
-  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  FOREIGN KEY(school_id) REFERENCES schools(id)
-);
-
-ALTER TABLE settings ADD CONSTRAINT "settings_unique" UNIQUE (type, value);
\ No newline at end of file
+ALTER TABLE settings ADD CONSTRAINT "settings_unique" UNIQUE (type, value);
```

Comment:
If we don't care about data when we deploy to prod, we could have this line inside the table right? Since we could create this constraint when the table is created

Reply:
@thepenguinco